### PR TITLE
Add .travis.yml for travis-ci

### PR DESCRIPTION
**Pull request by incyi**
I started to make a travis-ci integration, but could not finish it. Maybe someone wants to continue with this.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - cd $PROJECT_PATH #/build
+  #cmake configure
+  - cmake -DARM_NONE_EABI_TOOLCHAIN_PATH=/opt/gcc-arm-none-eabi-9-2019-q4-major -DNRF5_SDK_PATH=/opt/sdk/ -DUSE_OPENOCD=1
+  #build
+  - make -j pinetime-app
+
+notifications:
+  email:
+    recipients:
+      - inanc64+travis@gmail.com
```
Comment (incyi) [inline .travis.yml:37]: Change this or remove it.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # PineTime
+
+[![travis-ci build status](https://travis-ci.org/incyi/Pinetime.svg?branch=master)](https://travis-ci.org/incyi/pinetime)
```
Comment (incyi) [inline README.md:3]: Update url to JF002

File: README.md [hunk 3]
```diff
@@ -253,4 +256,4 @@ On Linux:
   * `trust <MAC ADDRESS>`
   * `pair <MAC ADDRESS>`
   
-Note that the current firmware only advertise for the first 3 minutes. If you cannot connect after more than 3 minutes, try resetting the device (push the button and hold it for 7-10 seconds).
```
Comment (incyi) [inline README.md:256]: Revert this.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  # install SDK in opt folder
+  - mkdir sdk
+  - cd sdk
+  - wget https://www.nordicsemi.com/-/media/Software-and-other-downloads/SDKs/nRF5/Binaries/nRF5SDK160098a08e2.zip
+  - unzip nRF5SDK160098a08e2.zip
+
+script:
+  - cd $PROJECT_PATH #/build
+  #cmake configure
+  - cmake -DARM_NONE_EABI_TOOLCHAIN_PATH=/opt/gcc-arm-none-eabi-9-2019-q4-major -DNRF5_SDK_PATH=/opt/sdk/ -DUSE_OPENOCD=1
```
Comment (incyi) [inline .travis.yml:30]: This fails on travis.

**Comment by JF002** [comment]
Hi, thanks for this PR.

I don't know Travis, is this a hosted CI that automatically builds the project when a new commit is pushed?

According to the build logs [here](https://travis-ci.org/github/incyi/pinetime), the build fails because it's trying to build using the AMD64 GCC from ubuntu instead of the one from the toolchain.

**Comment by incyi** [comment]
I made it working, but I did some cleaning so this pull request does not work anymore.

**Pull request closed without merging**